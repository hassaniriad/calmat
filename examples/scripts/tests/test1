//clear
i=3
u(i)=i
clear
