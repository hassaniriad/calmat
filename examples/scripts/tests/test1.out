(calmat) ~> i=3
(calmat) ~> u(i)=i
(calmat) ~> clear

i = 3

u = 0
    0
    3
